.gitignore

```diff
@@ -139,3 +139,8 @@ dmypy.json
 .DStore
 
 .vscode
+paper/paper_quarto_files/*
```

Comment:
```suggestion
# Paper rendering
paper/paper_quarto_files/*
```

Reply:
the rendering of the .qmd files makes a HTML, pdf, and tex file. Yep, no reason to include the tex file.